go.mod

```diff
@@ -32,3 +33,5 @@ require (
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 
 replace github.com/shirou/gopsutil => github.com/shirou/gopsutil v2.20.9+incompatible
+
+replace github.com/go-chi/chi/v4 => github.com/go-chi/chi v4.0.0+incompatible
```

Comment:
Note: this is needed due to https://github.com/DataDog/dd-trace-go/issues/1220 . We can remove it once the new dd-trace-go release is out.